threesixty/views.py

```diff
@@ -277,10 +277,16 @@ def get_question(self):
             raise models.Question.DoesNotExist('No question found.')
 
     def get_context_data(self, **kwargs):
+        qs = models.Question.objects
+        total_questions = qs.count()
+        answered_questions = answered = self.participant.answer_set.all().values_list('question_id', flat=True).count()
```

Comment:
Seems like `answered` is declared but is not used here.

So we can remove it, I guess, leaving only `answered_questions` declaration.
Also, `values_list` would not make a big difference here, so we can omit it.